designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRoot.kt

```diff
@@ -337,6 +347,7 @@ fun SquooshRoot(
                 customizationContext,
                 variantTransitions,
                 transitionedInteractionState ?: interactionState,
+                keyEventTracker,
```

Comment:
I think this means that during an animated transition, both sets of key events will be available and processed. I think maybe we should only have the target set (but don't think we need to make the change now)

Reply:
Good point, added https://github.com/google/automotive-design-compose/issues/1514